tzlocal/win32.py

```diff
@@ -121,7 +121,7 @@ def get_localzone_name():
     return _cache_tz_name
 
 
-def get_localzone():
+def get_localzone() -> tzinfo:
```

Comment:
This always returns a ZoneInfo, right?
```suggestion
def get_localzone() -> zoneinfo.ZoneInfo:
```

Reply:
Yeah, that's true, it will always be a ZoneInfo.